mathics/builtin/drawing/plot.py

```diff
@@ -416,9 +420,10 @@ def apply(self, functions, x, start, stop, evaluation, options):
                 functions_param[index] = f
             functions = functions.flatten_with_respect_to_head(SymbolList)
 
-        expr_limits = Expression(SymbolList, x, start, stop)
+        expr_limits = ListExpression(x, start, stop)
+        # FIXME: arrange forself to have a .symbolname property or attribute
         expr = Expression(
-            self.get_name(), functions, expr_limits, *options_to_rules(options)
+            Symbol(self.get_name()), functions, expr_limits, *options_to_rules(options)
```

Comment:
Maybe `get_head` can be used here?

Reply:
```
(trepan3k) self.get_head
** AttributeError: 'Plot' object has no attribute 'get_head'
```

(Re)read the FIXME  that appears on line 424.